src/convertToObject.js

```diff
@@ -5,8 +5,19 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+
+function convertToObject(stylesString) {
+  const stylesObject = {};
+
+  stylesString.split(';').forEach((style) => {
+    const [property, value] = style.split(':');
+
+    if (property && value) {
+      stylesObject[property.trim()] = value.trim();
+    }
+  });
+
+  return stylesObject;
 }
```

Comment:
The function `convertToObject` correctly splits the string and creates an object from the CSS declarations. However, it does not handle cases with multiple colons within the property values or multiple semicolons without any styles between them, which is acceptable as per the task description but could be improved for robustness.